src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
+    result: 0,
+
+    add(number) {
+      obj.result += number;
```

Comment:
```suggestion
      this.result += number;
```

Reply:
When I change 
      obj.result += number;   
on 
      this.result += number;
tests faild     
       TypeError: Cannot read property 'result' of undefined

      42 |
      43 |     add(number) {
    > 44 |       this.result += number;
         |       ^
      45 |
      46 |       return this;
      47 |     },